mobile/src/screens/chat-layout/chat-layout.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { type FC } from 'react';
+
+import { ScrollView, Text, View } from '#libs/components/components';
+import { useEffect, useRef, useState } from '#libs/hooks/hooks';
+
+import { ChatInput } from './components/chat-input/chat-input';
+import { MessageItem } from './components/message-item/message-item';
+import { PREVIOUS_USER } from './libs/constants';
+import { styles } from './styles';
+
+type Message = {
+  id: number;
+  isUser: boolean;
+  message: string;
+};
+
+const ChatLayout: FC = () => {
```

Comment:
its better to use React.FC not just FC when creating component (as there was discussion about this )